main/src/test/java/org/mobilitydata/gtfsvalidator/validator/RouteColorContrastValidatorTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import org.junit.Test;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.RouteColorContrastNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsRoute;
+import org.mobilitydata.gtfsvalidator.type.GtfsColor;
+import org.mockito.ArgumentCaptor;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+public class RouteColorContrastValidatorTest {
+
+    @Test
+    public void noRouteColorShouldNotGenerateNotice() {
```

Comment:
I agree. If we go strictly by the spec, the color contrast should be tested with the default colors when one of the 2 color fields is empty.

This notice should not be severe though (only a warning?). I imagine situations where this error could be counterproductive with the GTFS Grading Scheme. The notice should invite producers to provide the missing color, rather than update the color provided.